library/core/src/ptr/mut_ptr.rs

```diff
@@ -105,20 +106,21 @@ impl<T: ?Sized> *mut T {
     ///
     /// # Safety
     ///
-    /// While this method and its mutable counterpart are useful for
+    /// While this method and its [mutable counterpart] are useful for
```

Comment:
For symmetry I would then expect the same wording on that mutable counterpart -- something like "this method and its shared counterpart". Doesn't seem like that's what we got though? (It's a bit hard to tell in the GH diff.)